user_guide/user_guide.md

```diff
@@ -1 +1,61 @@
-t.b.d.
\ No newline at end of file
+## Requirements
+
+This package requires:
+* Python (>=3.8)
+* AWS CLI
+* AWS profile
+
+## Installation
+
+You can install the latest wheel package from the [Github Release page](https://github.com/exasol/script-languages-container-ci-setup/releases):
+`
+pip install https://github.com/exasol/script-languages-container-ci-setup/releases/download/$RELEASE/exasol_script_languages_container_ci-setup-$RELEASE-py3-none-any.whl
+`
+(Replace $RELEASE with the actual release you are interested in)
+
+## Usage
+
+There are actual 4 commands:
```

Comment:
4 ?

Maybe write something which is also not that prone to break on updates e.g.

`The following commands are available:`

Reply:
Done